pygmt/clib/conversion.py

```diff
@@ -241,7 +241,7 @@ def _as_array(vector):
 
     """
     if isinstance(vector, pandas.Series):
-        return vector.values
+        return vector.to_numpy()
```

Comment:
Actually, can't we just get rid of this `_as_array()` function?  Calling `np.asarray` on a `pandas.Series` returns a `numpy.array` anyway right? I don't see why this is even here.

Reply:
Yes, looks good to me. Not sure why `_as_array()` was designed like this.

Reply:
On another note, see https://www.pypandas.cn/en/blog/pandas-extension-arrays.html#the-current-state for why the change from `.values` to `.array` or `.to_numpy` has been made in Pandas.

Reply:
Looking at `git blame`, this code goes all the way back to December 2017 in #100. Back when pandas was at v0.21.

Reply:
Cool, I've remove the `_as_array()` function in favour of numpy's built-in `.asarray`.